src/algorithms/fardetectors/FarDetectorTrackerCluster.h

```diff
@@ -26,7 +26,7 @@ struct FDTrackerCluster {
   double energy{0.0};
   double time{0.0};
   double timeError{0.0};
-  std::vector<podio::ObjectID> rawHits;
```

Comment:
Can you add a comment as to why we use edm4hep, not podio (which we use everywhere else) with a link to relevant definition? I would hate for someone to change it back to podio in a year from now.

Also, can this be considered a bug in podio or edm4hep or edm4eic? One would expect these all to have consistent ObjectID types.

Reply:
I couldn't quite work out how this was wrong here but the edm4hep::ObjectID type has already been removed from the recent edm4hep releases. This means if we update, the old edm4hep::TrackerHit I'm using will break it anyway as it contains a vector of objectIDs rather than using podio relations. I believe our data model has progressed to a state the work around isn't necessary anymore.

Reply:
What's still missing from edm4eic is a Measurement3D, effectively Measurement2D swapping surface for the extra dimension, through the transformation to global coordinates (or not transforming back to 2D).